src/stories/tag/Tag.tsx

```diff
@@ -1,13 +1,21 @@
-import type { ComponentPropsWithoutRef, FC, ReactNode } from 'react';
+import type {
```

Comment:
Not relevant to the PR but could you add 'use client' at the top here?

Reply:
does it need `use client` — there's no interactivity here right?